### PR TITLE
Nerf click attack speed

### PR DESCRIPTION
**Pull request by anapple96**
The game has fallen into the Cookie Clicker trap and it's not a good thing.
Also buffed attack speed by a tiny bit to compensate

### CODE REVIEW

File: src/scripts/Battle.ts [hunk 1]
```diff
@@ -26,9 +26,9 @@ class Battle {
      */
     public static pokemonAttack() {
         // TODO: figure out a better way of handling this
-        // Limit pokemon attack speed, Only allow 1 attack per 900ms
+        // Limit pokemon attack speed, Only allow 1 attack per 750ms
         const now = Date.now();
-        if (this.lastPokemonAttack > now - 900) {
+        if (this.lastPokemonAttack > now - 750) {
```
Comment (RedSparr0w) [inline src/scripts/Battle.ts:31]: Why are we decreasing this?

Comment (anapple96) [reply src/scripts/Battle.ts:31]: "Also buffed attack speed by a tiny bit to compensate"

**Comment by Ishadijcks** [comment]
This seems like a knee jerk.
Also nerfing how often you can click is not nerfing click attack. You're nerfing autoclickers, and boosting attack for legit players.

**Comment by anapple96** [comment]
Yeah, that's the main purpose, the title is misleading

File: src/scripts/Battle.ts (hunk 1 above)
Comment (RedSparr0w) [reply src/scripts/Battle.ts:31]: This doesn't actually increase the pokemon attack speed though, It just allows people to cheat pokemon attack faster

Comment (anapple96) [reply src/scripts/Battle.ts:31]: I always forget the -900 in Battle.ts doesn't actually control the speed

Comment (anapple96) [reply src/scripts/Battle.ts]: Reverted

**Pull request closed without merging**